modules/neuroimaging/denoising/nlmeans/main.nf

```diff
@@ -0,0 +1,51 @@
+
+process DENOISING_NLMEANS {
+    tag "$meta.id"
+    label 'process_single'
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://scil.usherbrooke.ca/containers/scilus_2.0.0.sif':
+        'scilus/scilus:2.0.0' }"
+
+    input:
+    tuple val(meta), path(image), path(mask)
+
+    output:
+    tuple val(meta), path("*_denoised.nii.gz")      , emit: image
+    path "versions.yml"                             , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def args = []
+    if (mask) args += ["--mask $mask"]
```

Comment:
```suggestion
    def args = task.ext.args ?: ''
    def mask_arg = mask ? "--mask $mask" : ""
```
This makes it possible to add custom arguments to the script via the configuration file